style/menu.less

```diff
@@ -1,4 +1,6 @@
 .menu {
+	height: 160px;
```

Comment:
I kind of like the idea of adding an extra `mode` for popups, specially because this is something that maybe other components can use in the future. I just tested this with a 'halfscreen mode' (50vh), you can see that in this little sub-branch: https://github.com/wikimedia/wikipedia-kaios/commit/c925187f7728705bcba4d9d0e007b91c513688e9

But looking at the results, it seems setting the `height: 160px` directly in `.menu` might be simpler:

- Left column shows how it currently behaves with the `height: 160px` for both portrait and landscape (so without a `@media @landscape` media query). Portrait is in-line with [design mock up](https://app.zeplin.io/project/58dc46f4a83d1e477dd83859/screen/5fc68213375d92a6172724ae) and landscape is readable

- Right column shows the [halfscreen mode](https://github.com/wikimedia/wikipedia-kaios/commit/c925187f7728705bcba4d9d0e007b91c513688e9). Portrait looks basically the same but landscape does look smaller with `50vh`. We could add a media query in this case (`75vh`?) but then that's probably making it more complicated than needed.

Thoughts?

<img width="396" alt="image" src="https://user-images.githubusercontent.com/4752599/101666920-170f1f00-3a25-11eb-8864-0ef990fa24be.png">